browser/src/react-redux/App/User/updateUser.action.js

```diff
@@ -0,0 +1,14 @@
+export const updateUser = user => ({
```

Comment:
can we just use `setUser.action.js` here? Is there something that `User.action.js needs to do that `setUser.action.js` does not already do? I will change setUser.action.js to User.action.js. I can always change it back.

Reply:
This actions file will be meant for all actions regarding our authentication for the user. Just because we have a single action here, doesn't mean we won't want to add more actions in the future. In general, I think having a separate action file for every single action just clouds our feature folder with potentially too many files.